### PR TITLE
icu: add 69.1

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **icu/69.1**

---

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by madebr — DISMISSED** [review]

File: recipes/icu/all/conandata.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ sources:
   "68.2":
     url: "https://github.com/unicode-org/icu/releases/download/release-68-2/icu4c-68_2-src.tgz"
     sha256: "c79193dee3907a2199b8296a93b52c5cb74332c26f3d167269487680d479d625"
+  "69.1":
+    url: "https://github.com/unicode-org/icu/releases/download/release-69-1/icu4c-69_1-src.tgz"
+    sha256: "39ce83dd5d15c7539dde261733e106a391923f82caf1ce52ecaebb72d93b4579"
```
Comment (prince-chrismc) [inline recipes/icu/all/conandata.yml:22]:
```suggestion
    sha256: "4cba7b7acd1d3c42c44bb0c14be6637098c7faf2b330ce876bc5f3b915d09745"
```

**Review by prince-chrismc — COMMENTED** [review]
Someone changed the sha.....

File: recipes/icu/all/conandata.yml (hunk 1 above)
Comment (madebr) [reply recipes/icu/all/conandata.yml:22]: image

Comment (ericLemanissier) [reply recipes/icu/all/conandata.yml]: That's on me, I used the tag's archive instead of the release archive

Comment (madebr) [reply recipes/icu/all/conandata.yml]: Oh I thought the release managers fudged it.

**Review by madebr — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Pull request merged**